### PR TITLE
ES6 style changes and node v4.

### PR DESCRIPTION
**Pull request by jagoda**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "0.10"
-  - "4.0"
+  - 4
```
Comment (hueniverse) [inline .travis.yml]: keep `4.0` since we want to make sure we support that one specifically in addition to 4 latest.

Comment (jagoda) [inline .travis.yml]: Ah, ok.

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 language: node_js
 
 node_js:
-  - "0.10"
   - "4.0"
+  - "4"
+  - "5"
```
Comment (hueniverse) [inline .travis.yml]: Drop the "". No need.

Comment (jagoda) [inline .travis.yml]: Done.

**Pull request merged**